portafly/src/i18n/locales/en/strings.js

```diff
@@ -0,0 +1,48 @@
+export const OVERVIEW = {
```

Comment:
We are trying to reach an agreement on the format of the strings file.
At the moment, this is what is been discussed with Docs team.
It may change as the discussion continues

Reply:
If using a single file, the folder structure could be:
```
i18n
  locales
    en.json
    jp.json